src/redux/actions/user.js

```diff
@@ -0,0 +1,7 @@
+// import { createAction } from 'redux-actions'
+//
+// const SET_USER = 'SET_USER'
+//
+// const setUser = createAction(SET_USER)
+//
+// export { setUser }
```

Comment:
🤔🤔🤔🤔🤔

Reply:
as i wrote at the very beginning - code like this (comments) do nothing for us, so they are useless and we need to remove it